### PR TITLE
GUI: revalidate when toggling 'Keep Size' in AssembleDialog

### PR DESCRIPTION
**Pull request by Georgeto**
Loving your project, keep up the good work!

### CODE REVIEW

**Comment by mrexodia** [comment]
Currently on holiday so I'll merge this when I'm back (in a few days)...

**Comment by mrexodia** [comment]
@Georgeto could you explain what problem this fixes exactly?

File: src/gui/Src/Gui/AssembleDialog.cpp [hunk 1]
```diff
@@ -173,7 +173,7 @@ void AssembleDialog::on_lineEdit_textChanged(const QString & arg1)
 void AssembleDialog::on_checkBoxKeepSize_clicked(bool checked)
 {
     bKeepSizeChecked = checked;
-    mValidateThread->textChanged(ui->lineEdit->text());
```
Comment (Georgeto) [inline src/gui/Src/Gui/AssembleDialog.cpp]: `ValidateExpressionThread::textChanged()` only takes action if `text` has changed. So this call to `textChanged()` has no effect.

**Comment by Georgeto** [comment]
It revalidates the AssembleDialog when you toggle the _Keep Size_ checkbox.

keepsizeenabled
tooglekeepsize

As I later found out this feature was already implemented some time ago, but no longer working. (see my code comment)

**Comment by mrexodia** [comment]
Okay, thanks for the explanation!

**Pull request merged**